pkgs/development/tools/rust/cargo-inspect/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-inspect";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "mre";
+    repo = pname;
+    rev = version;
+    sha256 = "0rjy8jlar939fkl7wi8a6zxsrl4axz2nrhv745ny8x38ii4sfbzr";
+  };
+
+  cargoSha256 = "1pxvcf991w0jfxdissvwal5slrx7vpk3rqkzwk4hxfv0mjiqxsg5";
+
+  meta = with stdenv.lib; {
```

Comment:
If you're only using `stdenv.lib` you might as well just use `lib` directly here.

Reply:
Oh right! I must have picked it from somewhere else ^^ Thanks!